src/cli/subcommands/mpool_cmd.rs

```diff
@@ -32,6 +32,11 @@ pub enum MpoolCommands {
         #[arg(long)]
         from: Option<String>,
     },
+    /// Get the current nonce for an address
+    Nonce {
+        /// Address to check nonce for
+        address: String,
```

Comment:
I think `address: Address` works here since `Address` implements `FromStr`

Reply:
Oh yeah, actually it works. We have this rather backwards logic here that make it work. https://github.com/ChainSafe/forest/blob/e4548a87ebb897f5e74282939c063727c6f4d460/src/shim/address.rs#L178-L182